demisto_sdk/commands/common/tools.py

```diff
@@ -2008,11 +2008,43 @@ def get_latest_upload_flow_commit_hash() -> str:
     return last_commit
 
 
-def get_content_path() -> Path:
+def find_pack_folder(path: Path) -> Path:
+    """
+    >>> find_pack_folder(Path('root/Packs/MyPack/Integrations/MyIntegration/MyIntegration.yml'))
+    PosixPath('root/Packs/MyPack')
+    >>> find_pack_folder(Path('Packs/MyPack1/Scripts/MyScript/MyScript.py')).name
+    'MyPack1'
+    >>> find_pack_folder(Path('Packs/MyPack2/Scripts/MyScript')).name
+    'MyPack2'
+    >>> find_pack_folder(Path('Packs/MyPack3/Scripts')).name
+    'MyPack3'
+    >>> find_pack_folder(Path('Packs/MyPack4')).name
+    'MyPack4'
+    """
+
+    if "Packs" not in path.parts:
+        raise Exception(f"Could not find a pack for {str(path)}")
+    if path.parent.name == "Packs":
```

Comment:
no need to treat as a special case

Reply:
If we do not refer to this case, we get `PosixPath('/')`